recipes/si/all/conanfile.py

```diff
@@ -0,0 +1,77 @@
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+from conans.tools import Version
+
+import os
+
+
+class SiConan(ConanFile):
+    name = "si"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/bernedom/SI"
+    description = "A header only c++ library that provides type safety and user defined literals \
+         for handling pyhsical values defined in the International System of Units."
+    topics = ("physical units", "SI-unit-conversion",
+              "cplusplus-library", "cplusplus-17")
+    exports_sources = "CMakeLists.txt"
```

Comment:
`CMakeLists` wrapper is missing.

You should add one at the same level than conanfile.py, something like:

```cmake
cmake_minimum_required(VERSION 2.8.11)
project(cmake_wrapper)

include(conanbuildinfo.cmake)
conan_basic_setup()

add_subdirectory("source_subfolder")
```

Then, remove `source_folder=self._source_subfolder` when calling `self._cmake.configure`.
`no_copy_source = True` might need to be removed, not sure.